presto-benchto-benchmarks/README.md

```diff
@@ -67,6 +68,8 @@ macros:
 With the scene set up as in the previous section, the benchmark can be run with:
 ```bash
 ./mvnw clean package -pl presto-benchto-benchmarks
-java -Xmx1g -jar presto-benchto-benchmarks/target/presto-benchto-benchmarks-*-executable.jar \
+java -Xmx1g -jar path/to/benchto-driver.jar \
+    --sql presto-benchto-benchmarks/src/main/resources/sql \
+    --benchmarks presto-benchto-benchmarks/src/main/resources/benchmarks \
```

Comment:
I am not sure after all. It works here, but I can easily imagine one repository of SQL queries and benchmarks scattered across different directories (depending on use case, nightly, etc).
I mean we could remove this possibility, but I wouldn't want to add it later on (especially since we already separated profile file from overrides). The cost of having it is negligible. After all, we all complain about conventions in spring-boot, so why we should add it to benchto-driver (e.g: hardcoded names of `sql` and `benchmark` subfolders).